test/e2e/util.go

```diff
@@ -226,7 +226,7 @@ func createQueues(cxt *context) {
 }
 
 func deleteQueues(cxt *context) {
-	foreground := metav1.DeletePropagationForeground
+	foreground := metav1.DeletePropagationBackground
```

Comment:
should we wait for Queue deleted?

Reply:
Yes, this is not the reason. I found queue not deleted in the previous error log.

And found more error logs showing namespace `test` is being deleted when create it. Not sure why, we have already used foreground delete and also wait for namespace disappear